backend/python/app/services/implementations/comment_service.py

```diff
@@ -41,3 +41,47 @@ def create_comment(self, comment):
         db.session.commit()
 
         return new_comment
+
+    def update_comment(self, updated_comment):
+        try:
+            comment = Comment.query.filter_by(id=updated_comment.id).first()
+            if not comment:
+                raise Exception(
+                    "Comment with id={id} not found".format(id=updated_comment.id)
+                )
+            updated_comment.time = datetime.utcnow()
```

Comment:
hmmm I dont know if we want to ever change the time. If we do change the time, I dont think we'd want to change it if the comment was resolved but not updated  content-wise. I'll send a msg to the design channel and cc you